app/controllers/pages_controller.rb

```diff
@@ -29,9 +29,9 @@ class PagesController < ApplicationController
   def show
     @page = Page.find_by_page_name(params[:id]) || Page.find(params[:id])
     @news = News.published.limit(2) if params[:id] == 'index'
-    @blocks = @page.blocks.where(is_enabled: true).order(:name).paginate(
-      page: params[:page],
-      per_page: 9
-    )
+    # @blocks = @page.blocks.where(is_enabled: true).order(:name).paginate(
```

Comment:
Toto je ocividne nejaka vypnuta funkcionalita co ma aj admina. V trelle sa pytam evy ze ci to mozeme vyrazit uplne.